data/json/recipes/tools/tool.json

```diff
@@ -294,6 +294,26 @@
     "tools": [ [ [ "tongs", -1 ] ], [ [ "hotcut", -1 ] ] ],
     "components": [ [ [ "plastic_chunk", 2 ] ], [ [ "steel_lump", 2 ] ] ]
   },
+  {
+    "result": "forged_shears",
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "3 h",
+    "autolearn": true,
+    "proficiencies": [
+      { "proficiency": "prof_metalworking" },
+      { "proficiency": "prof_blacksmithing" },
+      { "proficiency": "prof_bladesmith" }
+    ],
+    "using": [ [ "forging_standard", 4 ] ],
+    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 3 }, { "id": "GRIND", "level": 2 } ],
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "hotcut", -1 ] ] ],
+    "components": [ [ [ "steel_lump", 3 ] ] ]
```

Comment:
Would you want some leather for wrapping the handles?

Reply:
I considered leather or long strings - but looking at the pics online - most of the don't have anything but metal. Possibly the wraps/handles don't survive heavy use, or make it uncomfortable...
![Uploading Screen Shot 2022-03-11 at 11.33.52.png…]()

Reply:
Not sure what's the problem, but couldn't upload any pics to the PR or a comment. Tried different browsers (FF and Chrome) - no avail.